### PR TITLE
[DOCS-3379] Modify instructions for attaching permissions to integrat…

### PR DESCRIPTION
**Pull request by buraizu**
### What does this PR do?
Modifies the section for attaching permissions to the existing AWS integration role.

### Motivation
DOCS-3379

### Preview
https://docs-staging.datadoghq.com/bryce/clarify-aws-role-modificiation-instructions/logs/log_configuration/archives/?tab=awss3#set-permissions

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by warrierr — APPROVED** [review]
LGTM

File: content/en/logs/log_configuration/archives.md
Comment (kayayarai) [inline content/en/logs/log_configuration/archives.md:122]:
```suggestion
1. [Create a policy][1] with the following two permission statements:
```

File: content/en/logs/log_configuration/archives.md
Comment (kayayarai) [inline content/en/logs/log_configuration/archives.md:124]:
```suggestion
```

File: content/en/logs/log_configuration/archives.md
Comment (kayayarai) [inline content/en/logs/log_configuration/archives.md:127]:
```suggestion
   * The `GetObject` and `ListBucket` permissions allow for [rehydrating from archives][2].
   * The `PutObject` permission is sufficient for uploading archives.
```
... to make this a sublist of the suggested ordered list

File: content/en/logs/log_configuration/archives.md
Comment (kayayarai) [inline content/en/logs/log_configuration/archives.md:156]:
```suggestion
```

File: content/en/logs/log_configuration/archives.md
Comment (kayayarai) [inline content/en/logs/log_configuration/archives.md:158]:
```suggestion
2. Attach the new policy to the Datadog integration role.
```

File: content/en/logs/log_configuration/archives.md
Comment (kayayarai) [inline content/en/logs/log_configuration/archives.md:164]:
```suggestion
   a. Navigate to **Roles** in the AWS IAM console.
   b. Locate the role used by the Datadog integration. By default it is named **DatadogIntegrationRole**, but the name may vary if your organization has renamed it. Click the role name to open the role summary page.
   c. Click **Add permissions**, and then **Attach policies**.
   d. Enter the name of the policy created above.
   e. Click **Attach policies**.

3. Edit the bucket names.
4. Optionally, specify the paths that contain your log archives.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
My edit suggests turning this into stepped instructions, which will make this review a bit of a mess (sorry!). I think it's clearer than summarizing and then going into details (ignore most of this review if you don't like it). Hopefully the steps hold up to scrutiny.

File: content/en/logs/log_configuration/archives.md
Comment (buraizu) [reply content/en/logs/log_configuration/archives.md]: @kayayarai thank you for this feedback. Within this context, it doesn't make sense to edit the bucket names or specify the log archive paths after the policy is attached, so I adjusted the order of these steps.

**Review by kayayarai — APPROVED** [review]
Adjusted the alignment of the code block and the bulleted list, so that they are nested beneath step 1. And resolved a merge conflict that was a PR from this morning. Just waiting on the preview build before it's good to merge.

**Pull request merged**
